dbt-athena-community/pyproject.toml

```diff
@@ -0,0 +1,104 @@
+[project]
+name = "dbt-athena-community"
+description = "The athena adapter plugin for dbt (data build tool)"
+readme = "../README.md"
+keywords = ["dbt", "adapter", "adapters", "database", "elt", "dbt-core", "dbt Core", "dbt Cloud", "dbt Labs", "athena"]
+requires-python = ">=3.8.0"
+authors = [
+    { name = "dbt Labs", email = "info@dbtlabs.com" },
+]
+maintainers = [
+    { name = "dbt Labs", email = "info@dbtlabs.com" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+# these versions should always match and they both should match the local version of dbt-athena
+dependencies = ["dbt-athena==1.8.5rc1"]
```

Comment:
Create a ticket to write a pre-commit hook that ensures these versions all match.

Reply:
https://github.com/dbt-labs/dbt-athena/issues/743